joschahenningsen/TUM-Live
# Bookmarks UI

### Motivation and Context
Bookmarks are a cool way to save _"klausurrelevantes"_. If many students use the bookmark feature we might even be able to generate a video section recommendations for the course admins from it.

### Description
- Adds bookmark UI to watch page

### Steps for Testing
1. Log in (Bookmarks should not be visible when not logged in!)
2. Navigate to a Livestream
3. Create a bookmark 
4. Edit a bookmark
5. Delete bookmarks by clicking on the `EDIT` button

### Screenshots

https://user-images.githubusercontent.com/29633518/201600630-0d89e363-78c1-49f0-ab64-6a09c0c7b4ef.mov
